extension/src/background/index.ts

```diff
@@ -23,4 +24,6 @@ export const initMessageListener = () => {
 
     return res;
   });
+
+  browser.runtime.onInstalled.addListener(migrateLocalStorageToBrowserStorage);
```

Comment:
run this the migration every time we install. This will also run when the browser auto upgrades the extension when there is a new version available